packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -31,6 +31,10 @@ const TEMPLATE_VERSIONS = require(`${ROOT_DIR}/lib/senders/emails/templates/_ver
 const SUBSCRIPTION_TERMS_URL = 'https://example.com/subscription-product/terms';
 const SUBSCRIPTION_PRIVACY_URL =
   'https://example.com/subscription-product/privacy';
+const SUBSCRIPTION_CANCELLATION_SURVEY_URL =
+  'https://survey.alchemer.com/s3/6534408/Privacy-Security-Product-Cancellation-of-Service-Q4-21';
```

Comment:
Hmm `DEFAULT_PRODUCT_DETAILS` may not be the best place for it, as we actually want to remove that as it could mask misconfigured products (see https://mozilla-hub.atlassian.net/browse/FXA-3846 -- which I just put into our maintenance sprint epic for more info -- thanks for the reminder).

Maybe just leave it where it is for now to avoid scope creep (exporting it though for the test). Can you file a follow-up for us to put this somewhere better going forward? Maybe add it as a task for the upcoming maintenance sprint?

We need to decide if this is an optional or a required part of a productConfig -- now that we have more than one product, should there be a default at all?